inference-engine/src/offline_transformations/src/moc_transformations.cpp

```diff
@@ -18,6 +18,13 @@
 #include <transformations/common_optimizations/convert_quantize_dequantize.hpp>
 #include <transformations/common_optimizations/pad_fusion.hpp>
 #include <transformations/common_optimizations/simplify_shape_of_sub_graph.hpp>
+#include <transformations/op_conversions/convert_scatter_elements_to_scatter.hpp>
+#include <transformations/common_optimizations/clamp_fusion.hpp>
```

Comment:
Looks like some transformation passes from these headers are not used. I didn't find `clamp`, `conv_to_binary_conv`

Reply:
Clamp fusion https://github.com/openvinotoolkit/openvino/pull/6749/files#diff-52fad3dbf19e0258bd6589e87538f71c88cdc58407a9aa1e956b9dd630907297R59
ConvToBinaryConv https://github.com/openvinotoolkit/openvino/pull/6749/files#diff-52fad3dbf19e0258bd6589e87538f71c88cdc58407a9aa1e956b9dd630907297R67

Reply:
There is no useless headers.